data/beaconrestapi/src/test/java/tech/pegasys/artemis/beaconrestapi/handlers/validator/GetValidatorDutiesTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.beaconrestapi.handlers.validator;
+
+import static javax.servlet.http.HttpServletResponse.SC_NO_CONTENT;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import io.javalin.http.Context;
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import tech.pegasys.artemis.api.ChainDataProvider;
+import tech.pegasys.artemis.provider.JsonProvider;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
We shouldn't need to use `MockitoExtension` because we're using the static `mock` method to create mocks.  We shouldn't actually have it even on the class path to force us to be consistent but somehow it snuck in...

Reply:
this is the way we currently use @Captor - without it that won't work. we can investigate how to fix that, but maybe not in this PR, as it's a pretty common pattern...